### PR TITLE
Add warning message for deprecated addDependencyTo option

### PR DESCRIPTION
**Pull request by RyanZim**
@ai Not sure if this is done right.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -63,6 +63,9 @@ function AtImport(options) {
         typeof options.addDependencyTo === "object" &&
         typeof options.addDependencyTo.addDependency === "function"
       ) {
+        console.warn(
```
Comment (michael-ciniawsky) [inline index.js]: But let @ai approve it first 😛

**Comment by ai** [comment]
Logic looks fine. But @michael-ciniawsky suggested better text.

**Comment by RyanZim** [comment]
@ai Will update text; you're fine with `console.warn()`?

**Comment by ai** [comment]
Yeap

**Comment by RyanZim** [comment]
@ai Fixed; had to do some pretty bad formatting to keep eslint happy.

**Pull request merged**